src/transformers/models/data2vec/configuration_data2vec.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+# Copyright 2018 The Google AI Language Team Authors and The HuggingFace Inc. team.
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" data2vec configuration"""
+from collections import OrderedDict
+from typing import Mapping
+
+from ...onnx import OnnxConfig
+from ...utils import logging
+from ..bert.configuration_bert import BertConfig
+
+
+logger = logging.get_logger(__name__)
+
+DATA2VEC_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "data2vec": "https://huggingface.co/data2vec/resolve/main/config.json",
+}
+
+
+class Data2VecConfig(BertConfig):
```

Comment:
```suggestion
class Data2VecConfig():
```

Could you please copy-paste all the code from `BertConfig` here? We are trying to not continue the abstraction from configs

Reply:
done!